src/test/java/teammates/test/cases/webapi/GetTimeZonesActionTest.java

```diff
@@ -43,10 +56,23 @@ protected void testExecute() throws Exception {
          * e.g. New York observes DST, so the offset is not always UTC-05:00 the entire year.
          * e.g. timezones can change, like Caracas modifying their timezone. This affects the offset as well.
          */
+
         assertEquals(8 * 60 * 60, output.getOffsets().get("Asia/Singapore").intValue());
-        assertEquals(-5 * 60 * 60, output.getOffsets().get("America/New_York").intValue());
-        assertEquals(11 * 60 * 60, output.getOffsets().get("Australia/Sydney").intValue());
-        assertEquals(0, output.getOffsets().get("Europe/London").intValue());
+        if (daylightSavingNewYork) {
+            assertEquals(-4 * 60 * 60, output.getOffsets().get("America/New_York").intValue());
+        } else {
+            assertEquals(-5 * 60 * 60, output.getOffsets().get("America/New_York").intValue());
```

Comment:
I understand what you're trying to do, but this is not the most elegant way to get around brittleness due to daylight savings. This will also cause problems in the future if `America/New_York` or any other time zone in this test decides to not observe daylight savings or change their time zone rules.

Instead, as I suggested in the issue, you can use a fixed offset time zone. For example:
```java
assertEquals(-5 * 60 * 60, output.getOffsets().get("Etc/GMT-5").intValue());
```

This can be guaranteed to never be brittle, because `Etc/GMT-5` is by definition five hours slower than GMT.

Please do this for the remaining test cases. We shouldn't rely on checking whether daylight savings is in effect, just use something that isn't affected by daylight savings instead.

Reply:
Thanks for helping me out! I have a question in regards to your example:
`assertEquals(-5 * 60 * 60, `output.getOffsets().get("Etc/GMT-5").intValue());`
If I am not mistaken, I think the argument of the get function should be "Etc/GMT+5" since the tz database has the signs reversed. I just wanted to check with you to make sure if this is correct.

Reply:
Yes, you are correct. My bad! Your current changes are correct.

For anyone following, this is the reason why. From inside the tzdb tables:
```
# We use POSIX-style signs in the Zone names and the output abbreviations,
# even though this is the opposite of what many people expect.
# POSIX has positive signs west of Greenwich, but many people expect
# positive signs east of Greenwich.  For example, TZ='Etc/GMT+4' uses
# the abbreviation "GMT+4" and corresponds to 4 hours behind UTC
# (i.e. west of Greenwich) even though many people would expect it to
# mean 4 hours ahead of UTC (i.e. east of Greenwich).
...
Zone	Etc/GMT-14	14	-	GMT-14	# 14 hours ahead of GMT
Zone	Etc/GMT-13	13	-	GMT-13
```